demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1586,9 +1586,10 @@ export class DemoMeetingApp
       this.toggleButton(button, isPromoted ? 'off' : 'disabled');
     }
 
-    // Additionally mute audio so it's not in an unexpected state when demoted
+    // Additionally mute audio and stop local video so it's not in an unexpected state when demoted
     if (!isPromoted) {
       this.audioVideo.realtimeMuteLocalAudio();
+      this.audioVideo.stopLocalVideoTile();
```

Comment:
unrelated demo fix :)

Reply:
Can we add a test case for this in the PR description :D

Reply:
too much pain to get integration tests to pass nowadays, I just let these changes sit in my local until i have a more critical PR :P